sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -4045,6 +4045,38 @@ goog.scope(function() {
     * @param {number} internalFormat
     * @param {number} width
     * @param {number} height
+    */
+    sglrReferenceContext.ReferenceContext.prototype.texImage2DDelegate = function (target, level, internalFormat, width, height) {
```

Comment:
Also, it's a pity we have to implement this twice. But the c++ class inheritance isn't implemented on the js side, with no sgrlContext implemented.